lms/djangoapps/teams/migrations/0005_add_composite_indexes.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+import pytz
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Create a composite index of course_id and is_active.
+        db.create_index('teams_courseteam', ['course_id', 'is_active'])
```

Comment:
It turns out that this composite index is not useful because is_active is almost always 1. I should remove it before merging the PR.